gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -6,6 +6,11 @@
 require 'linux_admin'
 
 module PostgresHaAdmin
+  RAILS_ROOT = [
```

Comment:
Can you put this inside the `FailoverMonitor` class so we are not defining top level constants?